scripts/screens/cat_screens.py

```diff
@@ -126,7 +126,7 @@ def bs_blurb_text(cat):
     if backstory != None and backstory in backstory_text:
         return backstory_text.get(backstory, "")
     else:
-        if cat.status in ['kittypet', 'loner', 'rogue', 'former clancat']:
+        if cat.status in ['kittypet', 'loner', 'rogue', 'former Clancat']:
```

Comment:
Is 'former Clancat' a valid status?
I thought 'exiled' is used for ex-Clancats.